src/view/menu.py

```diff
@@ -47,12 +52,25 @@ def draw(
             ofset_y = int(pos * 1.5 * self.button_dims[1])
             shifted_button_rect = self._button_rect.move(0, ofset_y)
             color = (
-                self.button_highlight
+                self.SELECTED_BUTTON_COLOR
```

Comment:
Constants have been added, but the old attributes weren't removed (`self.button_highlight`, `self.button_color` etc.). Also, use `Menu.SELECTED_BUTTON_COLOR` instead of `self`.